2.0.0_alpha04 Release

Hello guys,
is it somehow possible with the new alpha04 Release to disable the HTML Report (only for Paparazzi Snapshot tests) 
I currently have no chance to migrate to Gradle 9 and you enforce it.

I get some Crash with:
````
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: java.lang.NoClassDefFoundError: org/gradle/reporting/HtmlWriterTools
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.ClassPageRenderer$registerTabs$3.doExecute(ClassPageRenderer.kt:180)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.ClassPageRenderer$registerTabs$3.doExecute(ClassPageRenderer.kt:168)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.ErroringAction.execute(ErroringAction.kt:8)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PageRenderer$addTab$1.render(PageRenderer.kt:27)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PageRenderer$addTab$1.render(PageRenderer.kt:25)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.TabsRenderer.render(TabsRenderer.java:49)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PageRenderer.renderTabs(PageRenderer.kt:35)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PageRenderer$getContentRenderer$1.render(PageRenderer.kt:176)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PageRenderer$getContentRenderer$1.render(PageRenderer.kt:170)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.TabbedPageRenderer.render(TabbedPageRenderer.java:71)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.TabbedPageRenderer.render(TabbedPageRenderer.java:25)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.HtmlReportRenderer$DefaultHtmlReportContext$1.doExecute(HtmlReportRenderer.java:127)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.HtmlReportRenderer$DefaultHtmlReportContext$1.doExecute(HtmlReportRenderer.java:122)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.ErroringAction.execute(ErroringAction.java:36)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.IoActions$TextFileWriterIoAction.execute(IoActions.java:146)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.IoActions$TextFileWriterIoAction.execute(IoActions.java:126)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.IoActions.writeTextFile(IoActions.java:44)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.reporting.HtmlReportRenderer$DefaultHtmlReportContext.renderHtmlPage(HtmlReportRenderer.java:122)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at app.cash.paparazzi.gradle.reporting.PaparazziTestReporter$HtmlReportFileGenerator.run(PaparazziTestReporter.kt:132)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:67)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:60)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:167)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:60)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationExecutor$QueueWorker.execute(DefaultBuildOperationExecutor.java:161)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.runOperation(DefaultBuildOperationQueue.java:272)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.doRunBatch(DefaultBuildOperationQueue.java:253)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.lambda$runBatch$0(DefaultBuildOperationQueue.java:238)
2026-02-03T10:31:37.965+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.resources.AbstractResourceLockRegistry.whileDisallowingLockChanges(AbstractResourceLockRegistry.java:50)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.work.DefaultWorkerLeaseService.whileDisallowingProjectLockChanges(DefaultWorkerLeaseService.java:235)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.lambda$runBatch$1(DefaultBuildOperationQueue.java:238)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:263)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.work.DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:127)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.runBatch(DefaultBuildOperationQueue.java:224)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.operations.DefaultBuildOperationQueue$WorkerRunnable.run(DefaultBuildOperationQueue.java:192)
2026-02-03T10:31:37.966+0100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   ... 2 more
```

it looks like you use some function from the gradle 9.0 release.

I didn't find a way to workaround through this issue or to find a way to disable the HTML Report. (Without globally disabling it, also for unit tests) 

Do you have any other idea or did I miss some opportunity?

--- a/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ClassPageRenderer.kt
+++ b/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ClassPageRenderer.kt
@@ -5,7 +5,6 @@ import org.gradle.api.tasks.testing.TestOutputEvent
 import org.gradle.internal.html.SimpleHtmlWriter
 import org.gradle.internal.xml.SimpleMarkupWriter
 import org.gradle.reporting.CodePanelRenderer
-import org.gradle.reporting.HtmlWriterTools.addClipboardCopyButton
 import java.io.IOException
 
 internal class ClassPageRenderer(
@@ -175,7 +174,7 @@ internal class ClassPageRenderer(
               .characters("")
             resultsProvider.writeAllOutput(classId, TestOutputEvent.Destination.StdOut, htmlWriter)
             htmlWriter.endElement()
-            addClipboardCopyButton(htmlWriter, codeId)
+            addClipboardCopyButtonIfSupported(htmlWriter, codeId)
             htmlWriter.endElement()
           }
         }
@@ -196,11 +195,26 @@ internal class ClassPageRenderer(
               .characters("")
             resultsProvider.writeAllOutput(classId, TestOutputEvent.Destination.StdErr, htmlWriter)
             htmlWriter.endElement()
-            addClipboardCopyButton(htmlWriter, codeId)
+            addClipboardCopyButtonIfSupported(htmlWriter, codeId)
             htmlWriter.endElement()
           }
         }
       )
     }
+  }
+
+  private fun addClipboardCopyButtonIfSupported(htmlWriter: SimpleHtmlWriter, codeId: String) {
+    addClipboardCopyButtonMethod?.invoke(null, htmlWriter, codeId)
+  }
+
+  private companion object {
+    // Gradle 8 does not expose HtmlWriterTools, so copy buttons are best-effort.
+    val addClipboardCopyButtonMethod = runCatching {
+      Class.forName("org.gradle.reporting.HtmlWriterTools").getMethod(
+        "addClipboardCopyButton",
+        SimpleHtmlWriter::class.java,
+        String::class.java
+      )
+    }.getOrNull()
   }
 }

--- a/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ClassPageRenderer.kt
+++ b/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ClassPageRenderer.kt
@@ -207,6 +207,7 @@ internal class ClassPageRenderer(
     addClipboardCopyButtonMethod?.invoke(null, htmlWriter, codeId)
   }
 
+  // Workaround for Gradle 8.x users: remove once ported to the new testing infra
   private companion object {
     // Gradle 8 does not expose HtmlWriterTools, so copy buttons are best-effort.
     val addClipboardCopyButtonMethod = runCatching {

